database-plugins/src/main/java/io/cdap/plugin/db/connector/DBConnectorConfig.java

```diff
@@ -32,7 +33,7 @@
 /**
  * Defines a base {@link PluginConfig} that Database source, sink, and action can all re-use.
  */
-public class DBConnectorConfig extends PluginConfig {
+public class DBConnectorConfig extends PluginConfig implements DBConnectorProperties {
```

Comment:
what's the purpose of "DBConnectorProperties" interface ? don't see the usage of this interface.

Reply:
This is to tell the abstract connector what it can expect from the config, without this, from the config field, the connector is not able to get any information, with this, the connector is able to get the jdbc url or user name or password, the individual connector config can override this interface to provide them. For example, generic db will directly return the jdbc string, specific db will construct the jdbc url by host, port, etc.